pulp_python/app/models.py

```diff
@@ -5,23 +5,22 @@
 from django.core.exceptions import ObjectDoesNotExist
 from django.db import models
 from django.conf import settings
-from yarl import URL
-
 from pulpcore.plugin.models import (
     Content,
     Publication,
     Distribution,
     Remote,
-    Repository
+    Repository,
 )
+from pulpcore.plugin.responses import ArtifactResponse
```

Comment:
We could but we'd have to call it not-a-feature or the CI will be whine about it

Reply:
Is there a release date set for 3.22? We'd love to get this fix released since it's currently biting us but I appreciate Pulp's desire to not backport features.